js/booking.js

```diff
@@ -16,40 +16,43 @@ const challengeInput = multiStepForm.querySelector("#challenge");
 const header1 = multiStepForm.querySelector("#header1");
 const header2 = multiStepForm.querySelector("#header2");
 
+import { ApiArray as events } from "../main.js";
 
-import {ApiArray as events} from "../main.js";
+// global variable to get id from challenge button.
+let eventId;
+// goes through all classes of .open-button and gets rooms number."
+openModal.forEach((openModal) =>
```

Comment:
Have we fixed so this works when using filter?

Reply:
no not yet im working on it tho.